<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BSTerminalMainWindow</class>
 <widget class="QMainWindow" name="BSTerminalMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="mainTabBackground">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="horizontalFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="applicationBar" stdset="0">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>BlockSettle Terminal</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonUser">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
         <property name="usernameButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="currentIndex">
       <number>6</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="PortfolioWidget" name="widgetPortfolio">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="title">
        <string>OVERVIEW</string>
       </attribute>
      </widget>
      <widget class="RFQRequestWidget" name="widgetRFQ">
       <attribute name="title">
        <string>TRADE</string>
       </attribute>
      </widget>
      <widget class="RFQReplyWidget" name="widgetRFQReply">
       <attribute name="title">
        <string>DEALING</string>
       </attribute>
      </widget>
      <widget class="WalletsWidget" name="widgetWallets">
       <attribute name="title">
        <string>WALLETS</string>
       </attribute>
      </widget>
      <widget class="TransactionsWidget" name="widgetTransactions">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <attribute name="title">
        <string>TRANSACTIONS</string>
       </attribute>
      </widget>
      <widget class="ExplorerWidget" name="widgetExplorer">
       <attribute name="title">
        <string>EXPLORER</string>
       </attribute>
      </widget>
      <widget class="ChatWidget" name="widgetChat">
       <attribute name="title">
        <string>CHAT</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuBlockSettle">
    <property name="title">
     <string>BlockSettle</string>
    </property>
    <addaction name="actionAccount_Information"/>
    <addaction name="actionAuthentication_Addresses"/>
    <addaction name="actionEnter_Color_Coin_token"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About_BlockSettle"/>
    <addaction name="actionAbout_the_Terminal"/>
    <addaction name="action_Contact_BlockSettle"/>
    <addaction name="action_Version"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBlockSettle"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="action_Export_Transactions">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Export Transactions</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_Standard">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Standard</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Advanced">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Advanced</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_Expert">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Expert</string>
   </property>
  </action>
  <action name="action_Message_Signing_Verification">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Sign Message</string>
   </property>
  </action>
  <action name="action_EC_Calculator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;EC Calculator</string>
   </property>
  </action>
  <action name="actionBroadcast_Raw_Transaction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Broadcast Raw Transaction</string>
   </property>
  </action>
  <action name="actionView_Address_Book">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View &amp;Address Book</string>
   </property>
  </action>
  <action name="action_Sweep_Private_Key_Address">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Sweep Private Key/Address</string>
   </property>
  </action>
  <action name="action_Import_Private_Key_Address">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Import Private Key/Address</string>
   </property>
  </action>
  <action name="action_Create_New_Wallet">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Create New Wallet</string>
   </property>
  </action>
  <action name="action_Import_or_Restore_Wallet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Import or Restore Wallet</string>
   </property>
  </action>
  <action name="actionView_Address_Book_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>View &amp;Address Book</string>
   </property>
  </action>
  <action name="action_Fix_Damaged_Wallet">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Fix Damaged Wallet</string>
   </property>
  </action>
  <action name="actionLockbox_Manager">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Lockbox &amp;Manager</string>
   </property>
  </action>
  <action name="actionSimulfund_Promissory_Note">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Simulfund &amp;Promissory Note</string>
   </property>
  </action>
  <action name="actionSimulfund_Collect_Merge">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Simulfund &amp;Collect &amp;&amp; Merge</string>
   </property>
  </action>
  <action name="actionSimulfund_Review_Sign">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Simulfund &amp;Review &amp;&amp; Sign</string>
   </property>
  </action>
  <action name="action_About_BlockSettle">
   <property name="text">
    <string>&amp;About BlockSettle</string>
   </property>
  </action>
  <action name="actionVerify_Signed_Package">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Verify Signed Package</string>
   </property>
  </action>
  <action name="actionClear_All_Unconfirmed">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear All Unconfirmed</string>
   </property>
  </action>
  <action name="actionRescan_Databases">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rescan Databases</string>
   </property>
  </action>
  <action name="actionRebuild_and_Rescan_Databases">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rebuild and Rescan Databases</string>
   </property>
  </action>
  <action name="actionRescan_Balance">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rescan Balance</string>
   </property>
  </action>
  <action name="actionFactory_Reset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Factory Reset</string>
   </property>
  </action>
  <action name="actionAuthentication_Addresses">
   <property name="text">
    <string>Authentication Addresses</string>
   </property>
  </action>
  <action name="actionCreate_account">
   <property name="text">
    <string>Activate Market Data</string>
   </property>
  </action>
  <action name="actionApply_for_cash_trading">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Activate Blockchain products</string>
   </property>
  </action>
  <action name="actionApply_for_cash_coins_trading">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Activate Cash Products</string>
   </property>
  </action>
  <action name="actionContact_Support">
   <property name="text">
    <string>Contact Support</string>
   </property>
  </action>
  <action name="actionAccount_Information">
   <property name="text">
    <string>Account Information</string>
   </property>
  </action>
  <action name="actionDeposits">
   <property name="text">
    <string>Deposits</string>
   </property>
  </action>
  <action name="actionWithdrawal_Request">
   <property name="text">
    <string>Withdrawal Request</string>
   </property>
  </action>
  <action name="actionLink_Additional_Bank_Account">
   <property name="text">
    <string>Link Additional Bank Account</string>
   </property>
  </action>
  <action name="action_Sign_Message">
   <property name="text">
    <string>&amp;Sign Message</string>
   </property>
  </action>
  <action name="action_Verify_Message">
   <property name="text">
    <string>&amp;Verify Message</string>
   </property>
  </action>
  <action name="actionSweep_Addresses">
   <property name="text">
    <string>S&amp;weep Addresses</string>
   </property>
  </action>
  <action name="action_Offline_Transaction">
   <property name="text">
    <string>&amp;Offline Transaction</string>
   </property>
  </action>
  <action name="actionSetup_Local_Node">
   <property name="text">
    <string>Setup &amp;Local Node</string>
   </property>
  </action>
  <action name="action_One_time_Password">
   <property name="text">
    <string>&amp;One-Time Password</string>
   </property>
  </action>
  <action name="actionAccount_Management">
   <property name="text">
    <string>Account Management</string>
   </property>
  </action>
  <action name="actionAbout_the_Terminal">
   <property name="text">
    <string>About the &amp;Terminal</string>
   </property>
  </action>
  <action name="action_Contact_BlockSettle">
   <property name="text">
    <string>&amp;Contact BlockSettlle</string>
   </property>
  </action>
  <action name="action_Version">
   <property name="text">
    <string>&amp;Version</string>
   </property>
   <property name="toolTip">
    <string>Show version information</string>
   </property>
  </action>
  <action name="actionEnter_Color_Coin_token">
   <property name="text">
    <string>Equity Token Issuance</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletsWidget</class>
   <extends>QWidget</extends>
   <header>WalletsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransactionsWidget</class>
   <extends>QWidget</extends>
   <header>TransactionsWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PortfolioWidget</class>
   <extends>QWidget</extends>
   <header>PortfolioWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RFQRequestWidget</class>
   <extends>QWidget</extends>
   <header>RFQRequestWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RFQReplyWidget</class>
   <extends>QWidget</extends>
   <header>RFQReplyWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ExplorerWidget</class>
   <extends>QWidget</extends>
   <header>ExplorerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChatWidget</class>
   <extends>QWidget</extends>
   <header>ChatWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
